### ISSUE TITLE
Accessibility: Escape key should close the overlay

### ISSUE THREAD

**Issue by spadgett** [openshift/origin-web-catalog#326]
Suggested by @ncameronbritt, and I've had the same thought. I've tried to close the overlay with the escape key before, and it doesn't work. It's potentially an accessibility issue and inconsistent with our other Bootstrap modals.

Maybe we can optionally prompt if the user has typed something.

cc @openshift/team-ux-review

**Comment by beanh66** [comment]
@spadgett makes sense to me to allow users to press esc to close the wizard when changes have not been made.

**Comment by serenamarie125** [comment]
@jeff-phillips-18 should this behavior be built into the Angular PatternFly Overlay Modal?

**Comment by jeff-phillips-18** [comment]
@serenamarie125 yes

**Comment by spadgett** [comment]
@jeff-phillips-18 FYI, we have an `on-esc` directive that should make this relatively easy.

https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/util.js#L223-L234
